molo/forms/utils.py

```diff
@@ -207,13 +207,18 @@ def is_next_action(self, *actions):
     def is_end(self):
         return self.is_next_action(SkipState.END, SkipState.FORM)
 
-    def success(self, slug):
+    def success(self, slug, article):
```

Comment:
The article argument should default to `None` just in case

Reply:
I still think this should default to None in case this method is called from somewhere else.